### ISSUE TITLE
Is the source for these binaries public?

### ISSUE THREAD

**Issue by wcsmith** [StanfordSNR/vivace#1]
If so, we'd love to check it out. Great work!

**Comment by keithw** [comment]
Hello -- the authors have not released source code yet, but you can write to them: https://www.usenix.org/system/files/conference/nsdi18/nsdi18-dong.pdf

@nathanhjay

**Comment by keithw** [comment]
(For the record, I'm not a fan of having closed-source schemes in the Pantheon, because one of our big overall goals is to make these results easier to reproduce and understand by the broader community. But sometimes we have compromised on this to get interesting new schemes in here or to accommodate the needs of research we think is important or is done by our collaborators, e.g. in the cases of Vivace and FillP. I think these are the only exceptions we've made so far. I'm sympathetic though and hope these authors will be ready to release code in due time, especially as Vivace has been performing very well on the Pantheon recently!)
